File: DataBase/Models/OrganizationModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataBase.Models
{
    public class OrganizationModel
    {
        public string organizationID { get; set; }
        public string category { get; set; }
        public string organizationName { get; set; }
        public string organizationLongName { get; set; }
        public string accountingSystemName { get; set; }
        public string taxID { get; set; }
        public string organizationNPI { get; set; }
        public string ssn { get; set; }
        public string taxonomyCode { get; set; }
        public string capitationAccount { get; set; }
        public string claimsAccount { get; set; }
        public bool autoGenerateERA { get; set; }
        public AddressModel payToAddress { get; set; }
        public AddressModel billingAddress { get; set; }
    }
}

File: AngularMedicalGroup/src/app/models/organization.model.ts


export class Organization{
        public organizationID:string;
        public category:string;
        public organizationName:string;
        public organizationLongName:string;
        public accountingSystemName:string;
        public taxID:string;        
        public organizationNPI:string;
        public ssn:string;
        public taxonomyCode:string;
        public capitationAccount:string;
        public claimsAccount:string;
        public autoGenerateERA:string;
        public payToAddress:Address;
        public billingAddress:Address;
}
export class Address{
        public address1:string;
        public address2:string;
        public city:string;
        public state:string;
        public zip:string;
        public internationalZip:string;
        public countryDistict:string;
        public country:string;
        public phone:string;
        public faxNumber:string;
        public eMail:string;
        public contactPerson:string;
}